bin/fix_MultiRealmIPZenPack.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+##############################################################################
+# 
+# Copyright (C) Zenoss, Inc. 2018, all rights reserved.
+# 
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+# 
+##############################################################################
+
+ZP_NAME=MultiRealmIP
+FIXED_VERSION=2.2.4
+EGG=`find /opt/zenoss/ZenPacks/ -name ZenPacks.zenoss.${ZP_NAME}* | head -n 1`
+
+function version_gt() { test "$(printf '%s\n' "$@" | sort -V | head -n 1)" != "$1"; }
```

Comment:
I think `version_gt` returns the wrong result if FIXED_VERSION is 2.2.4 and CURRENT_VERSION is any value greater than 2.2.4, e.g.

```
$ test "$(printf '%s\n' "2.2.4" | sort -V | head -n 1)" != "2.2.5"; echo $?
0
```
In this scenario, the upgrade script will fail because the patch will fail.

Reply:
I checked it in this way:
```
[zenny@ip-10-111-23-147 ~]$ function version_gt() { test "$(printf '%s\n' "$@" | sort -V | head -n 1)" != "$1"; }
[zenny@ip-10-111-23-147 ~]$ FIXED_VERSION=2.2.4
[zenny@ip-10-111-23-147 ~]$ CURRENT_VERSION=2.2.3
[zenny@ip-10-111-23-147 ~]$ if version_gt $FIXED_VERSION $CURRENT_VERSION; then echo FIX_ZENPACK; fi
FIX_ZENPACK
[zenny@ip-10-111-23-147 ~]$ CURRENT_VERSION=2.2.4
[zenny@ip-10-111-23-147 ~]$ if version_gt $FIXED_VERSION $CURRENT_VERSION; then echo FIX_ZENPACK; fi
[zenny@ip-10-111-23-147 ~]$ CURRENT_VERSION=2.2.5
[zenny@ip-10-111-23-147 ~]$ if version_gt $FIXED_VERSION $CURRENT_VERSION; then echo FIX_ZENPACK; fi
[zenny@ip-10-111-23-147 ~]$ 
```